core/src/test/java/org/frankframework/mongodb/MongoDbSenderTest.java

```diff
@@ -56,14 +55,6 @@ public class MongoDbSenderTest extends SenderTestBase<MongoDbSender> {
 
 	private JndiMongoClientFactory mongoClientFactory;
 
-	@BeforeAll
```

Comment:
Dit is niet nodig, de annotaties zorgen al voor het opstarten van de container. Mappedport wordt verder ook niet gebruikt